.idea/libraries/Dart_SDK.xml

```diff
@@ -0,0 +1,26 @@
+<component name="libraryTable">
```

Comment:
Should exclude these files from git repo. Use `.gitignore` with a setup like below:
```
# See https://www.dartlang.org/tools/private-files.html

.DS_Store
Thumbs.db

# Files and directories created by pub
.packages
.pub/
.dart_tool/
build/
# If you're building an application, you may want to check-in your pubspec.lock | we don't
pubspec.lock

# Directory created by dartdoc
# If you don't generate documentation locally you can remove this line.
doc/api/

.atom
.idea
*.iml
.vscode

```